tests/try_scraper/test_invalid_revision.py

```diff
@@ -4,25 +4,16 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from mock import patch
-
+import pytest
 from mozdownload import TryScraper
 import mozdownload.errors as errors
-import mozhttpd_base_test as mhttpd
-
-
-class TestTryScraperInvalidParameters(mhttpd.MozHttpdBaseTest):
-    """test mozdownload TryScraper class with invalid parameters"""
-
-    @patch('mozdownload.treeherder.Treeherder.query_builds_by_revision')
-    def test_scraper(self, query_builds_by_revision):
-        """Testing download scenarios with invalid parameters for TryScraper"""
-        query_builds_by_revision.return_value = []
 
-        with self.assertRaises(errors.NotFoundError):
-            TryScraper(destination=self.temp_dir,
-                       base_url=self.wdir,
-                       logger=self.logger,
-                       platform='win32',
-                       revision='abc'
-                       )
+def test_invalid_parameters(httpd, tmpdir, mocker):
+    """Testing download scenarios with invalid parameters for TryScraper"""
+    query_builds_by_revision = mocker.patch('mozdownload.treeherder.Treeherder.query_builds_by_revision')
+    query_builds_by_revision.return_value = []
+    with pytest.raises(errors.NotFoundError):
+        TryScraper(destination=tmpdir,
+                   base_url=httpd.get_url(),
+                   platform='win32',
+                   revision='abc')
```

Comment:
Mind adding a final empty line to make the linter happy? Same for the other file.